core/src/main/java/nu/yona/server/crypto/pubkey/Encryptor.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2015, 2016 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
  *******************************************************************************/
-package nu.yona.server.crypto;
+package nu.yona.server.crypto.pubkey;
 
 public interface Encryptor
```

Comment:
Two variants is not good. Encrypt and Decrypt have Latin origins, so I prefer the Lattin "or" suffix. That's also pretty common, see [this link](https://books.google.com/ngrams/graph?content=encryptor%2C+encrypter&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cencryptor%3B%2Cc0%3B.t1%3B%2Cencrypter%3B%2Cc0).